src/app/pages/view/view-records/view-records.component.ts

```diff
@@ -84,21 +90,32 @@ export class ViewRecordsComponent implements OnInit {
         this.records = null;
 
         let datasetParams: any = {};
+        this.recordParams = {};
 
         if (this.projectId) {
             datasetParams['project'] = this.projectId;
         }
 
         if (this.dateStart) {
-            datasetParams['record__datetime__gte'] = this.dateStart.toISOString();
+            datasetParams['record__datetime__start'] = this.recordParams['datetime__start'] =
```

Comment:
The correct filter is now `record__datetime__gte` not 'record__datetime__start`. I've changed it last MR.

Reply:
Fixed